ush/config_defaults.yaml

```diff
@@ -2015,6 +2029,50 @@ task_run_post:
   USE_CUSTOM_POST_CONFIG_FILE: false
   CUSTOM_POST_CONFIG_FP: ""
   POST_OUTPUT_DOMAIN_NAME: '{{ workflow.PREDEF_GRID_NAME }}'
+  TESTBED_FIELDS_FN: ""
+
+#----------------------------
+# RUN PRDGEN config parameters
+#-----------------------------
+task_run_prdgen:
+  TN_RUN_PRDGEN: "run_prdgen"
+  NNODES_RUN_PRDGEN: 1
```

Comment:
I believe the default should be 22 since that was what was hardcoded in the command
```
mpiexec -np 22 --cpu-bind core cfp $CMDFILE  #Daniel: Move this somewhere else
```

Reply:
Yeah setting the default as 22 sounds good to me.